app/code/Magento/CatalogGraphQl/Model/Resolver/Products/DataProvider/CategoryTree.php

```diff
@@ -53,6 +54,11 @@ class CategoryTree
      */
     private $metadata;
 
+    /**
+     * @var ResolveInfo
+     */
+    private $resolverInfo;
```

Comment:
Rather than setting this as a class member, can we pass it via method arguments

Reply:
removed class member and used method argument